### ISSUE TITLE
Constructor GUI

### ISSUE THREAD

**Issue by delebota** [delebota/java-generator#3]
Prompt the user to select which variables they want in the constructor instead of just making an empty and full one.

**Comment by delebota** [comment]
Include option to call super();

**Comment by Michael-Thibault** [comment]
Yes please, to all of those GUI enhancements. Would certainly be handy.
